test/src/597-graft/main

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
usually I send the output of the publishing operation into a text file and then `grep` for the warning from there...